### PR TITLE
Add missing expects to specs

### PR DESCRIPTION
**Pull request by artofhuman**

### CODE REVIEW

File: spec/examples/schedule_spec.rb [hunk 1]
```diff
@@ -306,15 +306,15 @@
     it "should be equivalent to all_occurrences in terms of arrays" do
       schedule = IceCube::Schedule.new(Time.now, duration: IceCube::ONE_HOUR)
       schedule.add_recurrence_rule IceCube::Rule.daily.until(Time.now + 3 * IceCube::ONE_DAY)
-      schedule.all_occurrences == schedule.all_occurrences_enumerator.to_a
+      expect(schedule.all_occurrences).to eq(schedule.all_occurrences_enumerator.to_a)
```
Comment (pacso) [inline spec/examples/schedule_spec.rb:309]:
```suggestion
      expect(schedule.all_occurrences).to match_array(schedule.all_occurrences_enumerator.to_a)
```

File: spec/examples/schedule_spec.rb [hunk 2]
```diff
@@ -306,15 +306,15 @@
...
     end
   end
 
   describe :remaining_occurrences_enumerator do
     it "should be equivalent to remaining_occurrences in terms of arrays" do
       schedule = IceCube::Schedule.new(Time.now, duration: IceCube::ONE_HOUR)
       schedule.add_recurrence_rule IceCube::Rule.daily.until(Time.now + 3 * IceCube::ONE_DAY)
-      schedule.remaining_occurrences == schedule.remaining_occurrences_enumerator.to_a
+      expect(schedule.remaining_occurrences).to eq(schedule.remaining_occurrences_enumerator.to_a)
```
Comment (pacso) [inline spec/examples/schedule_spec.rb:317]:
```suggestion
      expect(schedule.remaining_occurrences).to match_array(schedule.remaining_occurrences_enumerator.to_a)
```

**Review by pacso — COMMENTED** [review]
Use array matcher

**Review by pacso — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed use of `delegate` method added in [66f1d797](https://github.com/ice-cube-ruby/ice_cube/commit/66f1d797092734563bfabd2132c024c7d087f683) , reverting to previous implementation. ([#522](https://github.com/ice-cube-ruby/ice_cube/pull/522))  by [@pacso](https://github.com/pacso)
 - Updated supported versions of Ruby and Rails and fixed standardrb lint issues. ([#552](https://github.com/ice-cube-ruby/ice_cube/pull/552)) by [@pacso](https://github.com/pacso)
 - Fixed `I18n.load_path` injection ([#546](https://github.com/ice-cube-ruby/ice_cube/pull/546)) by [@glaszig](https://github.com/glaszig)
+- Use `exepect` in specs ([#530](https://github.com/ice-cube-ruby/ice_cube/pull/530) by [@artofhuman](https://github.com/artofhuman))
```
Comment (lucascaton) [inline CHANGELOG.md:15]: Typo: `expect` 😉

**Pull request merged**